apps/examples/testcase/ta_tc/wifi_manager/itc/itc_wifi_manager_main.c

```diff
@@ -303,6 +319,430 @@ static void itc_wifi_manager_scan_ap_p(void)
+		TC_ASSERT_EQ("wifi_manager_disconnect_ap", ret, WIFI_MANAGER_SUCCESS);
+		WIFITEST_WAIT;
+		gettimeofday(&end_t, NULL);
+		average += diff_time(&start_t, &end_t);
+		sleep(5);
+		while (g_connect_status);
+	}
+
+	ret = wifi_manager_deinit();
+	TC_ASSERT_EQ("wifi_manager_deinit", ret, WIFI_MANAGER_SUCCESS);
+	printf("\n[Average disconnect time: %.2f ms]\n", average/(LOOP_SIZE*1000));
+	TC_SUCCESS_RESULT();
+}
+
+/**
+ * @testcase         itc_wifi_manager_success_ratio_ap
+ * @brief            Success ration init, join, leave, deinit
+ * @scenario
+ * @apicovered       wifi_manager_connect_ap, wifi_manager_disconnect_ap
+ * @precondition     none
+ * @postcondition    none
+ */
+
+static void itc_wifi_manager_success_ratio_ap(void)
+{
+	float init_cnt = 0.0f, s_init_cnt = 0.0f;
+	float join_cnt = 0.0f, s_join_cnt = 0.0f;
+	float leave_cnt = 0.0f, s_leave_cnt = 0.0f;
+	float deinit_cnt = 0.0f, s_deinit_cnt = 0.0f;
+
+	int i = 0;
+	wifi_manager_result_e ret = WIFI_MANAGER_FAIL;
+
+	wifi_manager_ap_config_s config;
+	config.ssid_length = strlen(CONFIG_EXAMPLES_TESTCASE_WIFI_MANAGER_ITC_SSID);
+	config.passphrase_length = strlen(CONFIG_EXAMPLES_TESTCASE_WIFI_MANAGER_ITC_PASSPHRASE);
+	strncpy(config.ssid, CONFIG_EXAMPLES_TESTCASE_WIFI_MANAGER_ITC_SSID, config.ssid_length + 1);
+	strncpy(config.passphrase, CONFIG_EXAMPLES_TESTCASE_WIFI_MANAGER_ITC_PASSPHRASE, config.passphrase_length + 1);
+	config.ap_auth_type = (wifi_manager_ap_auth_type_e)CONFIG_EXAMPLES_TESTCASE_WIFI_MANAGER_ITC_AUTHENTICATION;
+	config.ap_crypto_type = (wifi_manager_ap_crypto_type_e)CONFIG_EXAMPLES_TESTCASE_WIFI_MANAGER_ITC_CRYPTO;
+	printf("AP config: %s(%d), %s(%d), %d %d\n", config.ssid, config.ssid_length, config.passphrase, \
+			config.passphrase_length, config.ap_auth_type, config.ap_crypto_type);
+
+	for (i = 0; i < LOOP_SIZE; i++ ) {
+		ret = wifi_manager_init(&wifi_callbacks);
+		if (ret == WIFI_MANAGER_SUCCESS) {
+			s_init_cnt++;
+		} else {
+			init_cnt++;
+			continue;
+		}
+		init_cnt++;
+
+		ret = wifi_manager_set_mode(STA_MODE, NULL);
+		TC_ASSERT_EQ("wifi_manager_set_mode", ret, WIFI_MANAGER_SUCCESS);
+
+		ret = wifi_manager_connect_ap(&config);
+		if (ret == WIFI_MANAGER_SUCCESS)
+			s_join_cnt++;
+		join_cnt++;
+		WIFITEST_WAIT;
+		sleep(5);
+		ret = wifi_manager_disconnect_ap();
+
+		if (ret != WIFI_MANAGER_SUCCESS) {
+			printf("Wifi disconnect fails\n");
+		} else {
+			s_leave_cnt++;
+			WIFITEST_WAIT;
+		}
+		leave_cnt++;
+		sleep(5);
+		ret = wifi_manager_deinit();
+		if (ret == WIFI_MANAGER_SUCCESS) {
+			s_deinit_cnt++;
+		}
+		deinit_cnt++;
+	}
+
+	printf("\nSuccess Ratio \n");
+	printf("[Initialization Wifi] Success Ratio= %.2f%\n", (s_init_cnt/init_cnt)*100);
+	printf("[Connect Wifi]        Success Ratio= %.2f%\n", (s_join_cnt/join_cnt)*100);
+	printf("[Disconnect Wifi]     Success Ratio= %.2f%\n", (s_leave_cnt/leave_cnt)*100);
+	printf("[Deinitization Wifi]  Success Ratio= %.2f%\n", (s_deinit_cnt/deinit_cnt)*100);
```

Comment:
Put a space around(on each side of) most binary and ternary operators

    o example: = + -< > * / % | & ^ <= >= == != ? :

check for operators `/ `and `*` in your changes.